platform/broadcom/sonic-platform-modules-accton/debian/rules

```diff
@@ -14,6 +14,7 @@ include /usr/share/dpkg/pkg-info.mk
 export INSTALL_MOD_DIR:=extra
 
 PYTHON         ?= python2
```

Comment:
I see you're no longer building a Python 2 sonic-platform package. That's great. However, it appears you're still using Python 2 for something. Be aware that we will be removing Python 2 support from SONiC altogether in the near future.

Reply:
Ok. Due to this PR is for API 2.0, so I remove python 2. For others use python2.7, we will check and test for each platform.  After done, we will send another PR for each Accton platform.